source/agora/node/Registry.d

```diff
@@ -126,6 +129,7 @@ public class NameRegistry: NameRegistryAPI
 
     public void start ()
     {
+        this.client = this.zones[0].netman.makeRegistryClient();
```

Comment:
What's your idea about also using this for `redirect_primary`? This will limit configurability; i.e. currently this config field is not need to have `primary` zone capability but `redirect_primary` has.